test/unit/utils-locations.test.js

```diff
@@ -0,0 +1,34 @@
+const expect = require('chai').expect
+
+const {
+  locationHasExclusiveCollectionType,
+  locationHasResearchCenterPrefix
+} = require('../../lib/utils/locations')
+
+describe('utils/locations', () => {
+  describe('locationHasExclusiveCollectionType', () => {
+    it('returns true for location with matching collectionType', () => {
+      expect(locationHasExclusiveCollectionType('mal92', 'Research')).to.equal(true)
+      expect(locationHasExclusiveCollectionType('aga01', 'Branch')).to.equal(true)
+    })
+
+    it('returns false for location with non-matching collectionType', () => {
+      expect(locationHasExclusiveCollectionType('mal92', 'Branch')).to.equal(false)
+      expect(locationHasExclusiveCollectionType('aga01', 'Resaerch')).to.equal(false)
```

Comment:
though it doesn't matter for the test to pass, can you fix this typo for clarity of what is being tested here?